clientd3d/annotate.h

```diff
@@ -12,9 +12,9 @@
 #ifndef _ANNOTATE_H
 #define _ANNOTATE_H
 
-#define MAP_ANNOTATION_SIZE     (2 * FINENESS)    // Size of annotation indicator in FINENESS units
-#define MAP_ANNOTATION_MIN_SIZE 14                // Min size of drawn annotation in pixels
-
+#define MAP_ANNOTATION_RADIUS     FINENESS          // Radius of annotation indicator in FINENESS units
```

Comment:
This particular name is confusing (the others are OK).  The annotation is a square bitmap, so referring to a "radius" is super confusing, as it's the "radius of the inscribed circle", which no one cares about.  Mind putting just this one back?

Also, the comment could be clearer about this being the size of the source artwork (versus MAP_ANNOTATION_*_RADIUS being the size on screen, which is described in a comment).